tests/qa_automation/qa_run.pm

```diff
@@ -68,35 +68,34 @@ sub prepare_repos {
         assert_script_run "zypper --no-gpg-check -n ar -f '$qa_web_repo' qa-web";
     }
     else {
-        assert_script_run("echo 'info: No QA_WEB_REPO configured in this testsuit.'");
+        script_run("echo 'info: No QA_WEB_REPO configured in this testsuit.'");
     }
     assert_script_run "zypper --gpg-auto-import-keys ref";
-    assert_script_run("zypper -n in qa_testset_automation qa_tools", 300);
+    zypper_call("in qa_testset_automation qa_tools");
 }
 
 # Create qaset/config file, reset qaset, and start testrun
 sub start_testrun {
     my $self = shift;
     $self->create_qaset_config();
-    assert_script_run "/usr/share/qa/qaset/qaset reset";
+    assert_script_run("/usr/share/qa/qaset/qaset reset");
     my $testsuite = $self->test_suite();
-    assert_script_run "/usr/share/qa/qaset/run/$testsuite-run.openqa";
+    assert_script_run("/usr/share/qa/qaset/run/$testsuite-run.openqa");
```

Comment:
same style in whole file ... 

I prefer with () :) I can accept without .. but I really don't like mixed